### PR TITLE
Scheduler Customize Individual Views: rewrite

### PR DESCRIPTION
**Pull request by pavel-2020**
[Trello](https://trello.com/c/ggGmCBjS/1669-scheduler-demos-customization-3-demos)
[Demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Scheduler/CustomizeIndividualViews/jQuery/Light/)
CP: https://github.com/DevExpress/devextreme-demos/pull/967

### CODE REVIEW

File: JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md [hunk 1]
```diff
@@ -1 +1,4 @@
...
\ No newline at end of file
+To add a customized view, create an object in the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. In this object, you should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties that will override global Scheduler configuration.
+
+This demo has two customized views: *"week"* and *"workWeek"*. What differs them
+is that they group appointments by resources and set the [dateCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#dateCellTemplate). In addition, the *"workWeek"* view specifies the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) values.
```
Comment (AlexJameson) [inline JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md]:
```suggestion
This demo has two customized views: *"week"* and *"workWeek"*. They group appointments by resources and specify the [dateCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#dateCellTemplate). In addition, the *"workWeek"* view specifies the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) values.
```

File: JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md [hunk 2]
```diff
@@ -1 +1,4 @@
-In&nbsp;this demo, the _&laquo;Week&raquo;_ and _&laquo;Work Week&raquo;_ views of&nbsp;the Scheduler component are configured individually. The appointments are grouped by&nbsp;type (on&nbsp;the _&laquo;Week&raquo;_ view) and by&nbsp;priority (on&nbsp;the _&laquo;Work Week&raquo;_ view). In&nbsp;addition, the time window on&nbsp;the _&laquo;Work Week&raquo;_ view is&nbsp;narrowed down to&nbsp;working hours.
\ No newline at end of file
+To add a customized view, create an object in the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. In this object, you should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties that will override global Scheduler configuration.
```
Comment (AlexJameson) [inline JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md]: We can be much more straightforward here:

```suggestion
To customize a view, specify its settings in an object in the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. You should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties that override global Scheduler configuration.
```

File: JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md [hunk 3]
```diff
@@ -1 +1,3 @@
-In&nbsp;this demo, the _&laquo;Week&raquo;_ and _&laquo;Work Week&raquo;_ views of&nbsp;the Scheduler component are configured individually. The appointments are grouped by&nbsp;type (on&nbsp;the _&laquo;Week&raquo;_ view) and by&nbsp;priority (on&nbsp;the _&laquo;Work Week&raquo;_ view). In&nbsp;addition, the time window on&nbsp;the _&laquo;Work Week&raquo;_ view is&nbsp;narrowed down to&nbsp;working hours.
\ No newline at end of file
+To customize a view, configure its settings in an object of the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. You should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties that override global Scheduler settings.
```
Comment (AlexJameson) [inline JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md]: An object should be an element of the **views[]** array that means this object is **in** the array, right?

```suggestion
To customize a view, configure its settings in an object in the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. You should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties that override global Scheduler settings.
```

**Review by AlexJameson — DISMISSED** [review]

File: JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md (hunk 3 above)
Comment (RomanTsukanov) [inline JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md]:
```suggestion
To customize a view, configure its settings in an object of the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. You should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties to override global view settings.
```

File: JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md [hunk 4]
```diff
@@ -1 +1,3 @@
...
+
+This demo has two customized views: *"week"* and *"workWeek"*. They group appointments by resources and specify the [dateCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#dateCellTemplate). In addition, the *"workWeek"* view specifies the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) values.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md]:
```suggestion
This demo shows four views, two of which (Week and Work Week) are customized. Both customized views [group appointments](/Demos/WidgetsGallery/Demo/Scheduler/GroupOrientation/) by [resources](/Demos/WidgetsGallery/Demo/Scheduler/Resources/) and use the [dateCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#dateCellTemplate) to change the appearance of date cells. In addition, the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) properties are used to narrow down the time window on the Work Week view.
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

**Review by RomanTsukanov — DISMISSED** [review]

File: JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md [hunk 5]
```diff
@@ -1 +1,3 @@
...
\ No newline at end of file
+To customize a view, configure its settings in an object inside the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. You should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties to override global view settings.
+
+This demo shows four views, two of which (Week and Work Week) are customized. Both customized views [group appointments](/Demos/WidgetsGallery/Demo/Scheduler/GroupOrientation/) by [resources](/Demos/WidgetsGallery/Demo/Scheduler/Resources/) and use the [dateCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#dateCellTemplate) to change the appearance of date cells. In addition, the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) properties are used to narrow down the time window on the Work Week view.
```
Comment (Abadzhev) [inline JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md]: `... two of which ...` - sign of a complicated sentence. You can even put a period after `four views` and start the next sentence with `Week and Work Week views`. 

`are customized` - passive. 

'properties are used' - passive. 

The list of changes is written as a narrative, where I would suggest a more structured approach (a list). 

So here's the suggestion that fixes all those comments. 

```suggestion
This demo customizes two views - Week and Work Week - in the following manner: 

* Both views [group appointments](/Demos/WidgetsGallery/Demo/Scheduler/GroupOrientation/) by [resources](/Demos/WidgetsGallery/Demo/Scheduler/Resources/).
* Both views use the [dateCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#dateCellTemplate) to change the appearance of date cells.
* The Work Week view changes the [startDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour) properties to limit the time range.

Day and Month views use default settings.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**